docs/resources/conversations_messaging_integrations_instagram.md

```diff
@@ -0,0 +1,54 @@
+---
+page_title: "genesyscloud_conversations_messaging_integrations_instagram Resource - terraform-provider-genesyscloud"
+subcategory: ""
+description: |-
+  Genesys Cloud conversations messaging integrations instagram
+---
+# genesyscloud_conversations_messaging_integrations_instagram (Resource)
+
+Genesys Cloud conversations messaging integrations instagram
+
+## API Usage
+The following Genesys Cloud APIs are used by this resource. Ensure your OAuth Client has been granted the necessary scopes and permissions to perform these operations:
+
+* [POST /api/v2/conversations/messaging/integrations/instagram](https://developer.genesys.cloud/devapps/api-explorer#post-api-v2-conversations-messaging-integrations-instagram)
+* [GET /api/v2/conversations/messaging/integrations/instagram/{integrationId}](https://developer.genesys.cloud/devapps/api-explorer#get-api-v2-conversations-messaging-integrations-instagram--integrationId-)
+* [PATCH /api/v2/conversations/messaging/integrations/instagram/{integrationId}](https://developer.genesys.cloud/devapps/api-explorer#patch-api-v2-conversations-messaging-integrations-instagram--integrationId-)
+* [DELETE /api/v2/conversations/messaging/integrations/instagram/{integrationId}](https://developer.genesys.cloud/devapps/api-explorer#delete-api-v2-conversations-messaging-integrations-instagram--integrationId-)
+* [GET /api/v2/conversations/messaging/integrations/instagram](https://developer.genesys.cloud/devapps/api-explorer#get-api-v2-conversations-messaging-integrations-instagram)
+
+
+
+## Example Usage
+
+```terraform
+resource "genesyscloud_conversations_messaging_integrations_instagram" "test_sample" {
+  name                 = "Test Integration Instagram"
+  user_access_token    = "1234567890"
+  page_id              = "1"
+  messaging_setting_id = "2c4e3b8e-3c9f-45c9-82cd-4bb54c8f18f0"
+  supported_content_id = "019c37a7-ccb4-4966-b1d7-ddb20399f7ab"
+}
```

Comment:
Could you change these GUIDs for resource references? e.g. `genesyscloud_conversations_messaging_settings.settings.id` and `genesyscloud_conversations_messaging_supportedcontent.content.id`